src/message/bulkMessage.js

```diff
@@ -151,6 +151,25 @@ class BulkMessage{
         this.apiTemplate = value;
     }
 
+    /**
+     * Sets the ampBody for the message.
+     * (Optional) Either textBody or HtmlBody must be used or use a ApiTemplate with an AmpBody
+     * @param  {int} value
+     */
+    setAmpBody(value) {
+        if (typeof value === 'undefined' || !value) {
+          return;
+        }
+        if (!Number.isInteger(value)) {
```

Comment:
Type is string please check for type of string
`        if (typeof value !== 'string') {`
`          throw new Error("Invalid Amp Body, type of 'string' was expected.");`
`        }`